nebula/ui/components/VPNTutorialPopups.qml

```diff
@@ -0,0 +1,289 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.5
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import Mozilla.VPN 1.0
+import compat 0.1
+
+
+Item {
+
+    /*
+        TODOs
+
+        - Handle banner alerts
+        - Handle conflicts with update required
+
+        - Should tutorials timeout?
+        - Better UI in landscape mode
+        - Respond to targetElement y position changes
```

Comment:
would it be good if the C++ layer emits a new signal when the 'y' position changes? Or do you want to do it in QML? And... can it be done by QT connecting properties?